miniapps/meshing/mesh-optimizer.cpp

```diff
@@ -32,8 +32,22 @@
 // Compile with: make mesh-optimizer
 //
 // Sample runs:
-//   mesh-optimizer -m blade.mesh -o 2 -rs 0 -mid 2 -tid 1 -ni 20 -ls 2 -bnd
-//   mesh-optimizer -o 2 -rs 0 -ji 0.0 -mid 2 -tid 1 -lc 1e3 -ni 20 -ls 2 -bnd
+//   Blade shape:
+//     ./mesh-optimizer -m blade.mesh -o 4 -rs 0 -mid 2 -tid 1 -ni 200 -ls 2 -li 100 -bnd -qt 1 -qo 8
+//   Blade limited shape:
+//     ./mesh-optimizer -m blade.mesh -o 4 -rs 0 -mid 2 -tid 1 -ni 200 -ls 2 -li 100 -bnd -qt 1 -qo 8 -lc 5000
+//   ICF shape and equal size:
+//     ./mesh-optimizer -o 3 -rs 0 -mid 9 -tid 2 -ni 200 -ls 2 -li 100 -bnd -qt 1 -qo 8
+//   ICF shape and initial size:
+//     ./mesh-optimizer -o 3 -rs 0 -mid 9 -tid 3 -ni 100 -ls 2 -li 100 -bnd -qt 1 -qo 8
+//   ICF shape:
+//     ./mesh-optimizer -o 3 -rs 0 -mid 1 -tid 1 -ni 100 -ls 2 -li 100 -bnd -qt 1 -qo 8
+//   ICF limited shape:
+//     ./mesh-optimizer -o 3 -rs 0 -mid 1 -tid 1 -ni 100 -ls 2 -li 100 -bnd -qt 1 -qo 8 -lc 10
+//   ICF combo shape + size (rings, slow convergence):
+//     ./mesh-optimizer -o 3 -rs 0 -mid 1 -tid 1 -ni 1000 -ls 2 -li 100 -bnd -qt 1 -qo 8 -cmb
+//   3D pinched sphere shape (the mesh is in the mfem/data GitHub repository):
+//   * ./mesh-optimizer -m ../../../mfem_data/ball-pert.mesh -o 4 -rs 0 -mid 303 -tid 1 -ni 20 -ls 2 -li 500 -fix-bnd
```

Comment:
Why the `*`?

Reply:
To disable this run for `sample-runs.sh`, since the mesh file is in a separate repo.

Reply:
Actually, the `./` in front of the executable name is also blocking the sample runs from being extracted by `sample-runs.sh`.

Reply:
I think we should remove the `./`